addons/godot-doom-node/godot-doom-node.gdextension

```diff
@@ -1,8 +1,8 @@
 [configuration]
 
 entry_symbol = "godot_doom_library_init"
-compatibility_minimum = "4.1"
+compatibility_minimum = "4.3"
```

Comment:
FYI, I bumped the min version to `4.3`, as I synced with `godot-4.3-stable` for godot-cpp.

It should be possible to keep support for 4.1+ and just update to a newer commit from the `4.1` branch if the fixes for recent SCons errors have been backported.